docs/datastore/neo4j_graph.md

```diff
@@ -0,0 +1,94 @@
+# Setup and configure Neo4j
+
+## Before you begin
+
+1. Make sure you have a GoogleCloud project and billing is enabled.
+
+1. Set your `PROJECT_ID` environment variable:
+
+    ```bash
+    export PROJECT_ID=<YOUR_PROJECT_ID>
+    ```
+
+1. [Install](https://cloud.google.com/sdk/docs/install) the gcloud CLI.
+
+1. Set gcloud project:
+
+    ```bash
+    gcloud config set project $PROJECT_ID
+    ```
+
+1. Make sure you have python version 3.11+ installed.
```

Comment:
You still need to enable the AIPlatform API, see the other instructions for the command